rpcs3/Emu/RSX/rsx_utils.h

```diff
@@ -5,6 +5,8 @@ extern "C"
 #include <libavutil/pixfmt.h>
 }
 
+#include <cmath>
```

Comment:
Should maybe probably add it to stdafx.h instead? Also your branch needs to be rebased.